enterprise/server/util/redisutil/redisutil.go

```diff
@@ -39,6 +39,12 @@ var (
 	cacheShardedRedisPassword   = flag.String("cache.redis.sharded.password", "", "Redis password")
 	cacheRedisMaxValueSizeBytes = flag.Int64("cache.redis.max_value_size_bytes", 0, "The maximum value size to cache in redis (in bytes).")
 
+	// Remote Execution Redis
+	remoteExecRedisTarget          = flag.String("remote_execution.redis_target", "", "A Redis target for storing remote execution state. Falls back to app.default_redis_target if unspecified. Required for remote execution. To ease migration, the redis target from the cache config will be used if neither this value nor app.default_redis_target are specified.")
+	remoteExecRedisShards          = flagutil.StringSlice("remote_execution.sharded_redis.shards", []string{}, "Ordered list of Redis shard addresses.")
+	remoteExecShardedRedisUsername = flag.String("remote_execution.sharded_redis.username", "", "Redis username")
+	remoteExecShardedRedisPassword = flag.String("remote_execution.sharded_redis.password", "", "Redis password")
+
```

Comment:
I might be a bit late to the party (since I see other flags are already defined here) but I think it'd be nice to keep `redisutil` buildbuddy-agnostic:

* Instead of defining `RemoteExecutionRedisClientConfig` et. al. here, maybe define it in the place where we construct the remote execution redis client (I guess that'd be `enterprise/server/cmd/server/main.go`)
* For the default redis config which is used in a bunch of places, we could use the new pattern of defining a separate library for it, maybe `enterprise/server/util/redisconf` or something like that?

wdyt?

(this PR LGTM otherwise -- it'd be fine to address this in another PR)

Reply:
Moved these out.